pass-model-ext-nihms/src/main/java/org/dataconservancy/pass/model/ext/nihms/NihmsSubmission.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataconservancy.pass.model.ext.nihms;
+
+import org.dataconservancy.pass.model.Submission;
+
+/**
+ * 
+ * @author Karen Hanson
+ * @version $Id$
+ */
+public class NihmsSubmission extends Submission {

```

Comment:
I think that this'll need to be reconciled with the existing NIHMS [submission model](https://github.com/OA-PASS/nihms-submission/tree/master/nihms-model) in `https://github.com/OA-PASS/nihms-submission/tree/master/nihms-model`.  At least, I'd like to understand how we expect these two models to interact.

Reply:
Yeah - I haven't settled on how to deal with these model extensions. The idea here is that NIHMS submission records in Fedora specifically need a PMID field associated with them as it's the only field that is consistently on every record in the NIHMS spreadsheets. Other kinds of Submission records do not need this and shouldn't have one.  So when I do createResource, I will pass in this NihmsSubmission as a different flavor of Submission specifically used by the NIHMS loader and will expect to be able to look up this value later. Maybe there is a better way to handle these... or maybe PMID should be part of the core model...